src/datasets/containers.jl

```diff
@@ -38,6 +38,8 @@ function loadfile(file::String)
         return FileIO.load(file, view = true)
     elseif endswith(file, ".csv")
         return DataFrame(CSV.File(file))
+    elseif endswith(file, ".txt")
+        return String(read(file))
```

Comment:
```suggestion
        return read(file, String)
```
Per last signature in https://docs.julialang.org/en/v1/base/io-network/#Base.read.